projects/pgai/pgai/vectorizer/embeddings.py

```diff
@@ -659,3 +665,123 @@ async def _fallback_retry_individually(
                         )
                     )
         return embeddings
+
+
+class VoyageAI(ApiKeyMixin, BaseModel, Embedder):
+    """
+    Embedder that uses Ollama to embed documents into vector representations.
+
+    Attributes:
+        implementation (Literal["ollama"]): The literal identifier for this
+            implementation.
+        model (str): The name of the Ollama model used for embeddings.
+        truncate (bool): Truncate input longer than the model's context length
```

Comment:
nit: update doc. it's talking about ollama

Reply:
done